CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
 # Changelog
 
+## v1.9.0 (2020-8-6)
+
+### Enhancements
+
+#### 🌟 Multi Window Support (Beta)
+
+TestCafe can now automate user actions in multiple windows. You can switch between open windows during the test. Make edits in one window and check that the other window's content changes dynamically to reflect these modifications.
+
+![Testing in multiple windows](docs/articles/images/blog/2020-07-23-multi-window.gif)
+
+When the main window opens a child window, TestCafe automatically switches to this new window and continues test actions there:
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Login page`
+    .page('https://login.wrike.com/login/');
+
+const googleButton = Selector('div.login-panel-footer__login-with > button');
+
+test('Login via Google', async t => {
+    await t
+        .click(googleButton)
+        .typeText('input[type=email]', 'This text will be entered inside the pop-up');
+});
+```
+
+You can use the [t.openWindow](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/openwindow.html) method to open a child window in test code:
+
+```js
+import { Selector, ClientFunction } from 'testcafe';
+
+fixture `Test page`
+    .page('https://devexpress.github.io/testcafe/example/');
+
+test('Open a new window', async t => {
+    await t.openWindow('http://example.com');
+
+    const url = await t.eval(() => document.documentURI);
+
+    await t.expect(url).eql('http://example.com');
+});
+```
+
+The [t.switchToWindow](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/switchtowindow.html) method enables you to switch between open windows. You can use a window identifier or a predicate to specify the window that should be activated:
```

Comment:
> You can use a window identifier or a

In code and [typings](https://github.com/DevExpress/testcafe/commit/c864b535f5feae2a34f29671e324c155c8e23dfd#diff-3f3d3966c089c8767e453bd78a2a7b77R85), we use the `window descriptor` term. Consider using this term here.

Reply:
Thanks, forgot to update this, fixed